phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewsConfigurationLibrary.java

```diff
@@ -39,7 +39,11 @@
 
     @Bean
     public ViewLibrary viewLibrary() {
-        return new ViewLibrary(applicationName, themeConfiguration.themeManager(), phoenicisGlobalConfiguration.objectMapper());
+        return new ViewLibrary(
```

Comment:
My best guess is, that the problem is inside the formatter configuration.
@plata and me, we updated the formatter plugin version in #767, because the old version didn't do its job with the indention. Maybe the new plugin version requires a newer version of the formatter configuration.